polus-vector-label-plugin/src/requirements.txt

```diff
@@ -0,0 +1,4 @@
+
+scipy==1.5.2
+zarr ==2.5.0
+numba==0.51.2
```

Comment:
I forgot to mention in my review, please include the version of bfio this code works with. Although the container already comes with bfio, people cloning this repo won't know they need to use bfio until they try to run the code and an error is generated.